lib/wadler/print.rb

```diff
@@ -4,30 +4,38 @@
 module Oppen
   # Wadler.
   class Wadler
+    # Config containing customization flags.
     attr_reader :config
+    # The current indentation amount.
     attr_reader :current_indent
-    attr_reader :space
+    # The new line String.
     attr_reader :new_line
+    # Container of the output.
     attr_reader :out
+    # The indentator.
+    attr_reader :space
+    # The tokens list that is being built.
     attr_reader :tokens
+    # The whitespace character. Used to trim trailing whitespaces.
     attr_reader :whitespace
+    # Maximum line width desired.
     attr_reader :width
```

Comment:
Yeah I am still not sure whether we need to add spaces between different `attr_*` definitions.

When reading here, I think it might make more sense to compact the definitions, and avoid the introduction of new lines. They can take up too much vertical space.